himawari_nav.f90

```diff
@@ -3,6 +3,7 @@
 ! *    Copyright (C) 2016-2019 Simon Proud <simon.proud@physics.ox.ac.uk>
 ! *    License: CC BY-NC-ND 4.0
 ! *
+!  2020/5/29 CP changed integers to floats to avoid Aritmetic overflow error in gfortran
```

Comment:
Could you remove this line? All the changes are documented through github, so it's unnecessary.